src/engines/kotor/module.cpp

```diff
@@ -91,7 +91,7 @@ Module::Module(::Engines::Console &console)
 		  _pcPositionLoaded(false),
 		  _inDialog(false),
 		  _cameraHeight(0.0f) {
-
+	loadTexturePack();
```

Comment:
Ah, damn, so that model needs the texture pack after all? Annoying :P 

Could you do that texturepack loading in `Game::mainMenu()` instead, right before MainMenu is instantiated? With a comment that the Malak model needs the texture pack?